app/src/assets/localization/en/app_settings.json

```
@@ -9,6 +9,7 @@
   "__dev_internal__quickTransferExportPython": "Enable Python Export for Quick Transfer",
   "__dev_internal__reactQueryDevtools": "Enable React Query Devtools",
   "__dev_internal__reactScan": "Enable React Scan",
+  "__dev_internal__protocolContentsLog": "Enable Protocol Contents Logging",
```

does ODD not have a description for the ffs? if not, can you make this string a bit more descriptive? like the following:
```suggestion
  "__dev_internal__protocolContentsLog": "Enable QT Protocol Contents Logging",
```